Snippets/scale-font.py

```diff
@@ -0,0 +1,273 @@
+"""
+Script to change UnitsPerEm of a Font
+USAGE:
+scale-font.py path/to/inputFont.ttf targetUPM path/to/outputFont.ttf
+
+Only works on .ttf (no CFF) for now
+"""
+
+from fontTools import ttLib
+from sys import argv
+
+def scaleValue_toUPM(value, UPM, target_UPM):
```

Comment:
i don't like the mixing of camelCase and snake_case, try to stick to one or the other (I prefer the latter, but in fontTools there's probably more of the former).

Reply:
yes I quite like snake_case as well to list the arguments implicitly